File: LV0/5명씩.js
function solution(names) {
  let answer = [];
  let count = 0;
  for (let i = 0; i < names.length; i++) {
    count++;
    if (count === 1) {
      answer.push(names[i]);
    }
    if (count === 5) {
      count = 0;
    }
  }
  return answer;
}

File: LV0/5명씩.java
import java.util.*;

class Solution {

  public String[] solution(String[] names) {
    ArrayList<String> answer = new ArrayList<>();
    int count = 0;
    for (int i = 0; i < names.length; i++) {
      count++;
      if (count == 1) {
        answer.add(names[i]);
      }
      if (count == 5) {
        count = 0;
      }
    }
    return answer.toArray(new String[0]);
  }
}
